src/components/BuySellForm/BuySellPaymentSection/BuySellPaymentSection.tsx

```diff
@@ -67,6 +68,20 @@ const BuySellPaymentSection = ({
                     )}
                 </Text>
                 <div className='flex gap-[0.8rem] flex-wrap'>
+                    {advertiserPaymentMethods?.map((paymentMethod, index) => {
+                        const isSelected = selectedPaymentMethodIds.includes(Number(paymentMethod.id));
+
+                        return (
+                            <PaymentMethodCard
+                                isDisabled={isDisabled || (disableAdvertiserPaymentMethods && !isSelected)}
+                                key={index}
+                                medium
+                                onSelectPaymentMethodCard={onSelectPaymentMethodCard}
+                                paymentMethod={paymentMethod}
+                                selectedPaymentMethodIds={selectedPaymentMethodIds}
+                            />
+                        );
+                    })}
```

Comment:
to guarantee all payment methods are shown, map over advertiserPaymentMethods since they should be able to select any of their created payment methods